guides/common/attributes-titles.adoc

```diff
@@ -7,6 +7,7 @@
 :ContentManagementDocTitle: Managing content
 :ConvertingHostRHELDocTitle: Converting a host to RHEL
 :DeployingAWSDocTitle: Deploying {ProjectName} on Amazon Web Services
+:HammerDocTitle: Hammer CLI reference
```

Comment:
@apinnick I would like to have your opinion.

I want to change the title, but I'm not sure whether to call it _Hammer CLI reference_ or _Using Hammer CLI_. In downstream, we put it into the Reference category, but it contains procedures as well. What do you think?

Reply:
For d/s, I would prefer "Hammer CLI guide" so that it is consistent with the existing "API Guide" (not sure why that title capitalizes "guide" -- will check). 

I would not use the contents  alone for determining the title. All API and CLI guides contain procedures for installing, configuring, setting log levels, etc. 

API and tool guides tend to be named either "Using the 'x' tool" or "X reference guide". Personally, I do not think it matters which format is used as long as the guide names are consistent. You could change the API guide to "Using the Satellite REST API" and call this guide "Using the Hammer CLI tool". :-)  

Consistency is the most important consideration, in my opinion.

Reply:
I intend to change the title of the API guide too #3212 . I don't like having "guide" in the title, when we're avoiding it everywhere else.

Reply:
+1 I also don't like "guide" in the title.